deep-learning/_posts/2017-08-31-polynet.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: review
+title: "PolyNet: A Pursuit of Structural Diversity in Very Deep Networks"
+tags: deep-learning CNN classification layers
+author: "Philippe Poulin"
+cite:
+    authors: "X. Zhang, Z. Li, C. C. Loy, D. Lin"
+    title:   "PolyNet: A Pursuit of Structural Diversity in Very Deep Networks"
+    venue:   "CVPR 2017"
+pdf: "https://arxiv.org/pdf/1611.05725.pdf"
+---
+
+## Summary
+Generalization of Inception-ResNet modules.  A PolyInception module is represented as a polynomial composition of Inception blocks, which may share parameters inside the same module.
+
+
+![](/deep-learning/images/polynet/figure3.png)
+
+
+## PolyInception modules
+
+**Notation:**
+- I is Input
+- $$F, G, H$$ are inception "blocks"
```

Comment:
What is the difference between F, G and H ? 
H doesn't seem to be used anywhere afterwards?

Reply:
1. Each letter represents an inception block with different parameters. If a letter appears twice, it means that the parameters are shared between the blocks.

2. You're right, I forgot to include the 3rd order equations where $H$ is used.